multi/config_m2.lua

```diff
@@ -1,9 +1,13 @@
+nodes_load = require("config_load_nodes")
+
 box.cfg {
     listen = 3014,
-    wal_dir = 'm2/xlog',
-    snap_dir = 'm2/snap',
+    wal_dir = 'm2',
+    snap_dir = 'm2',
```

Comment:
Excuse me, was this change important for this PR?

Reply:
Sorry, missed it. Cleaned from commit.